www/lobby/id.ts

```diff
@@ -56,7 +56,24 @@ lobby_id_router.get('/:id', async (req: Request, res: Response) => {
   const { lobby } = verified;
   if (!lobby.participants.includes(userId)) return res.status(406).json('User is not part of the lobby').end();
 
-  res.status(200).json(lobby.getClientResponse());
+  const clientRes = lobby.getClientResponse();
+  const participants = await Promise.all(clientRes.participants.map(async (id : string) => {
```

Comment:
should we store these on the database so we dont need to call a bunch of requests?

Reply:
i think we should have it so any time we add a new user in the `Lobby` object, we update the metadata there

Reply:
Should we just recall synthesize playlist? since songs r probably going to get updated too